components/marketing_signup_modal/index.jade

```diff
@@ -18,7 +22,7 @@
     .marketing-signup-modal-error
   .marketing-signup-modal-divider Or
   a.avant-garde-button-black.marketing-signup-modal-fb(
-    href=sd.AP.facebookPath
+    href=sd.AP.facebookPath + '?acquisition_initiative=' + sd.MARKETING_SIGNUP_MODAL_SLUG
```

Comment:
This is related to [this PR](https://github.com/artsy/artsy-passport/pull/88#discussion_r90137532) where we pass the acquisition initiative through to the server-side.